### ISSUE TITLE
Starting models does not work: "...checkpoint file in .bin/.pt/.safetensors format could not be located:"

### ISSUE THREAD

**Issue by 8skull** [oobabooga/text-generation-webui#3769]
### Describe the bug

This text appears when I try to load any model:

2023-08-31 17:18:19 ERROR:The model could not be loaded because its checkpoint file in .bin/.pt/.safetensors format could not be located.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Open oobabooga and loading any model.

### Logs

```shell
C:\users\user1\1\oobabooga_windows\installer_files\env\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117.dll
2023-08-31 17:17:47 INFO:Loading the extension "gallery"...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
2023-08-31 17:18:19 INFO:Loading stable-vicuna-13B-GPTQ...
2023-08-31 17:18:19 ERROR:The model could not be loaded because its checkpoint file in .bin/.pt/.safetensors format could not be located.
```

### System Info

```shell
RTX 4070
32 RAM
CPU: i7 newest generation (13th?)
```

**Comment by 8skull** [comment]
Please help. 

Can it have something to do with that I installed the program manually later, I did in fact not delete that as I worked alot on manually installing it and handpicking to install torch and other things to pip.

Btw, I have Python 3.10.6 instead, which was recommended to me.

**Comment by Jake36921** [comment]
try cloning the model repository from huggingface to the models folder and also putting the model inside it.

**Comment by Jake36921** [comment]
Yes, download all the files in the model repository. using **git clone** is easier than downloading individually
![image](https://github.com/oobabooga/text-generation-webui/assets/113159896/497b5b85-87cb-4979-bcb9-b73e02dd81d6)
